gateway/gateway.go

```diff
@@ -27,12 +29,12 @@ type IPCGatewayState struct {
 }
 
 type Subnet struct {
-	ID             SubnetID
+	ID             ipcsdk.SubnetID
 	Stake          abi.TokenAmount
 	TopDownMsgs    cid.Cid // TCid<TAmt<CrossMsg, CROSSMSG_AMT_BITWIDTH>>,
 	Nonce          uint64
 	CircSupply     abi.TokenAmount
-	Status         Status
+	Status         ipcsdk.Status
 	PrevCheckpoint Checkpoint
 }
```

Comment:
Why does the subnet type live at the gateway package?

Reply:
specifically, at `gateway/gateway.go` when it could be part of `gateway/types.go`?

Reply:
No strong reason, will change it